File: TicTacToeAPI/Models/Game.cs
namespace TicTacToeAPI.Models;
public class Game
{
    public int Id { get; set; }
    public required string Board { get; set; }
    public required string NextPlayer { get; set; }
    public required string Winner { get; set; }
    public TicTacToeGameState GameState { get; set; }
}

File: TicTacToe.UI/src/app/models/game.ts
export class Game {
    id?: number;
    board = '';
    nextPlayer = '';
    winner = '';
    gameState?: GameState;
}

export enum GameState {
    XMove = 'XMove',
    OMove = 'OMove',
    XWin = 'XWin',
    OWin = 'OWin',
    Draw = 'Draw'
}